latent_variable_model.py

```diff
@@ -292,6 +297,7 @@ def forward(self, z, receptive_field_centers):
             # coeffs shape: 1 x 1 x H x {1 if shared, else N} x 1
             coeffs = self.coeffs[None, None, :, :, None]
             response = torch.sum(response * coeffs, dim=2)  # B x L x N x E
+            response = response + receptive_scale[None, None]
```

Comment:
Should we not introduce this parameter for the 'separate' and the 'bump' as well?

Reply:
For the 'separate' case, it doesn't make sense to include it, since each neuron has a unique spline defined by their own coeff-weights. As for the 'bump', I agree, we could absolutely include it there.

Reply:
That would allow for the 'bump' variant to all share a "common bump", with both a scaling and a shift option, but in that case they wouldn't all look like the same gaussian bump any more. But if that's a feature we wish to include (making it even more similar to the 'shared' variant), then I'll go ahead and update it.

Reply:
It's not a big issue if it is superfluous in the bump case. I think I would just add it to all versions..

Reply:
I'll add it to the 'bump' then. A bit more hesitant to add it to the 'separate' case, since the parameter would be redundant, but if over-specification is not an issue, I suppose it's okay..?